src/groupeddataframe/splitapplycombine.jl

```diff
@@ -8,6 +8,8 @@ const MULTI_COLS_TYPE = Union{AbstractDataFrame, NamedTuple, DataFrameRow, Abstr
 # we do not use nothing to avoid excessive specialization
 const NOTHING_IDX_AGG = Int[]
 
+isemptyVectorInt(@nospecialize x) = x isa Vector{Int} && isempty(x)
```

Comment:
TBH I'd find it clearer to repeat `x isa Vector{Int} && isempty(x)` (or just `x isa Vector && isempty(x)`) each time we need this. If you prefer keeping this, call it `isempty_vector_int`. :-)